l10n/en/products/vpn/platforms/desktop.ftl

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+# Meta strings
+vpn-desktop-page-title = { -brand-name-mozilla-vpn } for desktop (PC & Laptop)
+vpn-desktop-page-description = Get fast, easy security on your desktop and laptop computer through { -brand-name-mozilla }, one of the most trusted names in technology.
+
+# Desktop strings
+vpn-desktop-hero-headline = Secure your internet connection with { -brand-name-mozilla-vpn } for desktop
+
+# Variables:
+#   $devices (number) - number of available devices
+vpn-desktop-ability = { -brand-name-mozilla-vpn } gives you the ability to connect confidently and keep your information safe from prying eyes and data thieves when you’re online. { -brand-name-mozilla-vpn } uses advanced { -brand-name-wireguard }® protocol to encrypt personal data on your laptop or desktop computer — for up to { $devices } devices. Stream shows, play games, shop, and go about your daily life online knowing that you’re secure.
+
+vpn-desktop-maker-of-firefox = As the maker of the { -brand-name-firefox } web browser and one of the most trusted fighters for internet privacy, { -brand-name-mozilla } is committed to ensuring you feel safe and respected every time you go online.
+
+vpn-desktop-wifi-headline = Feel safe using public wifi
+vpn-desktop-wifi-copy = Our VPN shields your internet connection from hackers and spies, so you can shop from the coffee shop, check your bank account from the train and go about your online business anywhere without safety concerns.
+
+vpn-desktop-about-headline = About { -brand-name-mozilla }
+vpn-desktop-about-copy = Best known for our { -brand-name-firefox-browser }, { -brand-name-mozilla } is a global community of users, contributors and developers dedicated to keeping the power of the internet in people's hands. Founded in 1998, we’re one of the pioneers of the free and open web — and it’s our mission to keep it that way.
```

Comment:
Should be curly quote in `people’s hands`

Reply:
Updating now.